software-modules/langs/java/javac/java.sh

```diff
@@ -36,8 +35,50 @@ savechanges /tmp/$NAME.hsm
 ## Clean package to maintain only relevant files
 hsm2dir /tmp/$NAME.hsm
 cd /tmp/$NAME.hsm
-find . ! -path "./usr/lib/jvm*" ! -path "./usr/bin*" ! -path "./usr/share/icons*" ! -path "./usr/share/man*" ! -path "./usr/share/pixmaps*" ! -path "./etc/alternatives*" ! -path "./usr/share/application-registry*" ! -path "./usr/share/lintian*" ! -path "./usr/share/applications/java-documentation.desktop" ! -path "./usr/share/doc/reference/java*" -delete
+
+# Array to store paths to exclude that contain java things
+declare -a EXCLUDE_PATHS=(
```

Comment:
I think this method is more alike to "The paths I mention have things requried to the module, erase everything else" rather than "Erase these specific folders and keep the rest", I think that might leave leftover files from other system things.